tools/travis-before-install.sh

```diff
@@ -5,12 +5,17 @@ free -m
 df -h
 ulimit -a
 
-if [ -n "$PPC64_LE" ]; then
+if [ -n "$DOWNLOAD_OPENBLAS" ]; then
   pwd
   ls -ltrh
   target=$(python tools/openblas_support.py)
-  sudo cp -r $target/64/lib/* /usr/lib
-  sudo cp $target/64/include/* /usr/include
+  if [ -d "$target/usr/local" ]; then
+      sudo cp -r $target/usr/local/lib/* /usr/lib
+      sudo cp $target/usr/local/include/* /usr/include
+  else
+      sudo cp -r $target/64/lib/* /usr/lib
+      sudo cp $target/64/include/* /usr/include
+  fi
```

Comment:
This seems awkward. Could we add an argument to `openblas_support.py:__main__` to  print another level in the directory, maybe `--get-root` or so?

Reply:
Well, the openblas archive contents are a bit of a mess.
Made it to strip the common prefix directory instead.